.vsts-dotnet.yml

```diff
@@ -173,10 +173,8 @@ extends:
             signType: $(SignType)
             zipSources: false
             feedSource: https://devdiv.pkgs.visualstudio.com/_packaging/MicroBuildToolset/nuget/v3/index.json
-            ${{ if eq(variables['System.TeamProject'], 'DevDiv') }}:
+            ${{ if and(eq(variables['SignType'], 'real'), eq(variables['System.TeamProject'], 'DevDiv')) }}:
               ConnectedPMEServiceName: 6cc74545-d7b9-4050-9dfa-ebefcc8961ea
-            ${{ else }}:
-              ConnectedPMEServiceName: 248d384a-b39b-46e3-8ad5-c2c210d5e7ca
           env:
```

Comment:
what does it mean that System.TeamProject not equals DevDiv, when does that happen

Reply:
oh you changed it